calibrations/models.py

```diff
@@ -15,8 +15,16 @@ def target_is_in_season(self, query_date: datetime = datetime.utcnow()):
     """
     seasonal_start = int(self.targetextra_set.filter(key='seasonal_start').first().float_value)
     seasonal_end = int(self.targetextra_set.filter(key='seasonal_end').first().float_value)
+    current_month = query_date.month
```

Comment:
This section is causing a conflict. It's easy to resolve. Basically, the changes were added in both dev and main. So, git doesn't know how to handle it. 

If you merge dev into main locally via command line, you should be able to see this conflict and just delete one version of this code. Be sure to catch all of the `<<<<<` and `=====` bits too.

When you've done that just git add the file and commit the merge again.

Reply:
I also needed to abort the initial merge. Without doing that, the post-correction merge attempt failed with the message: "fatal: You have not concluded your merge (MERGE_HEAD exists)."